queries/powershell/highlights.scm

```diff
@@ -0,0 +1,323 @@
+; Punctuation
+;------------
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+(hash_literal_expression
+  [
+    "@{"
+    "}"
+  ] @punctuation.special)
+
+(array_expression
+  [
+    "@("
+    ")"
+  ] @punctuation.special)
+
+(member_access
+  "." @punctuation.special)
```

Comment:
I think this should also be removed once the broader `"."` capture is added to `@punctuation.delimiter`

Reply:
Yes, it seems redudant after adding `"."` to delimiters